pkg/middleware/http/tokenBucket.go

```diff
@@ -0,0 +1,31 @@
+package http
+
+import (
+	"sync"
+
+	"github.com/bnb-chain/greenfield-storage-provider/pkg/log"
+	"golang.org/x/time/rate"
+)
+
+type BandWidthLimiterConfig struct {
+	Enable bool
+	R      rate.Limit
+	B      int
```

Comment:
Add more commentaries to explain the config meaning. 
This naming convention is a bit vague for the name that the user needs to configure, and the user may not understand it.

Reply:
Annotated